tests/components/MegadraftEditor_test.js

```diff
@@ -598,7 +598,6 @@ describe("MegadraftEditor Component", () => {
     const toolbar = this.wrapper.find(Toolbar);
     // editor is undefined :-/
     //expect(toolbar.prop("editor")).to.equal(this.component.refs.editor);
-    expect(toolbar.prop("actions")).to.equal(this.component.actions);
```

Comment:
Can you please keep this assertion? The toolbar still needs to receive `actions`, the only difference is that `this.component.actions` doesn't exist anymore.